secrets_management/csharp/sdk/order-processor/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Dapr.Client;
+
+const string DAPR_SECRET_STORE = "localsecretstore";
+const string SECRET_NAME = "secret";
+var client = new DaprClientBuilder().Build();
+
+// Get secret from a local secret store
+var secret = await client.GetSecretAsync(DAPR_SECRET_STORE, SECRET_NAME);
+Console.WriteLine($"Fetched Secret: {string.Join(", ", secret)}");
```

Comment:
Joining inside a string interpolation expression is a bit unusual.  typically you'd put the literals in the base string and only dynamically add the varible.  Something more like this:

`Console.WriteLine($"Fetched Secret: {secret}");`